tests/cli_test.py

```diff
@@ -380,7 +381,7 @@ def abi_file_with_nodeos_test():
 
         node.processCleosCmd('set abi eosio.token ' + malicious_token_abi_path, 'set malicious eosio.token abi', returnType=ReturnType.raw)
 
-        cmdArr = node._Node__transferFundsCmdArr(accounts[2], accounts[3], '25.0000 SYS', 'm', False, None, False, False, 90, False)
+        cmdArr = node._Transactions__transferFundsCmdArr(accounts[2], accounts[3], '25.0000 SYS', 'm', False, None, False, False, 90, False)
```

Comment:
If `__transferFundsCmdArr` is useful outside of the class, can we make it public by not having the `__` prefix?

Reply:
Addressed in